scripts/deploy.py

```diff
@@ -0,0 +1,296 @@
+    }
+
+
+def check_funds(stakeholders_accounts):
+    a = stakeholders_accounts
+    fundsMissing = 0
+    for accountName, requiredAmount in REQUIRED_FUNDS.items():
+        if a[accountName].balance() >= REQUIRED_FUNDS[accountName]:
+            print('{} funding ok'.format(accountName))
+        else:
+            fundsMissing += REQUIRED_FUNDS[accountName] - a[accountName].balance()
+            print('{} needs {} but has {}'.format(
+                accountName,
+                REQUIRED_FUNDS[accountName],
+                a[accountName].balance()
+            ))
+    
+    if fundsMissing > 0:
+        if a[INSTANCE_OPERATOR].balance() >= REQUIRED_FUNDS[INSTANCE_OPERATOR] + fundsMissing:
+            print('{} sufficiently funded to cover missing funds'.format(INSTANCE_OPERATOR))
+        else:
+            print('{} needs additional funding of {} to cover missing funds'.format(
+                INSTANCE_OPERATOR,
+                REQUIRED_FUNDS[INSTANCE_OPERATOR] + fundsMissing - a[INSTANCE_OPERATOR].balance()
+            ))
+
+
+def amend_funds(stakeholders_accounts):
+    a = stakeholders_accounts
+    for accountName, requiredAmount in REQUIRED_FUNDS.items():
+        if a[accountName].balance() < REQUIRED_FUNDS[accountName]:
+            missingAmount = REQUIRED_FUNDS[accountName] - a[accountName].balance()
+            print('funding {} with {}'.format(accountName, missingAmount))
+            a[INSTANCE_OPERATOR].transfer(a[accountName], missingAmount)
+
+
+def deploy(
+    stakeholders_accounts, 
+    publishSource=False
+):
+
+    # define stakeholder accounts
+    a = stakeholders_accounts
+    instanceOperator=a[INSTANCE_OPERATOR]
+    instanceWallet=a[INSTANCE_WALLET]
+    oracleProvider=a[ORACLE_PROVIDER]
+    chainlinkNodeOperator=a[NODE_OPERATOR]
+    riskpoolKeeper=a[RISKPOOL_KEEPER]
+    riskpoolWallet=a[RISKPOOL_WALLET]
+    investor=a[INVESTOR]
+    productOwner=a[PRODUCT_OWNER]
+    insurer=a[INSURER]
+    customer=a[CUSTOMER1]
+    customer2=a[CUSTOMER2]
+
+    # token definition, funding of investor and customer
+    erc20Token = TestCoin.deploy({'from': instanceOperator})
+
+    # gif instance deployment
+    instance = GifInstance(instanceOperator, instanceWallet=instanceWallet)
+    instanceService = instance.getInstanceService()
+    instanceOperatorService = instance.getInstanceOperatorService()
+    componentOwnerService = instance.getComponentOwnerService()
+
+    # ayii deployment
+    ayiiDeploy = GifAyiiProductComplete(instance, productOwner, insurer, oracleProvider, chainlinkNodeOperator, riskpoolKeeper, investor, erc20Token, riskpoolWallet)
+
+    ayiiProduct = ayiiDeploy.getProduct()
+    ayiiOracle = ayiiProduct.getOracle()
+    ayiiRiskpool = ayiiProduct.getRiskpool()
+
+    product = ayiiProduct.getContract()
+    oracle = ayiiOracle.getContract()
+    riskpool = ayiiRiskpool.getContract()
+
+    # investor funding and bundle creation
+    bundleInitialFunding=1000000

```

Comment:
move to constant

Reply:
no, prefer to see the actual value where it is used in this case